https://bugs.gentoo.org/303120

make sure the linux version constructor runs before the libproc constructor
since the latter uses variables setup by the former

fix by Chris Coleman

Index: proc/version.c
===================================================================
RCS file: /cvsroot/procps/procps/proc/version.c,v
retrieving revision 1.7
diff -u -p -r1.7 version.c
--- a/proc/version.c	9 Feb 2003 07:27:16 -0000	1.7
+++ b/proc/version.c	14 Nov 2010 00:22:44 -0000
@@ -33,7 +33,7 @@ void display_version(void) {
 
 int linux_version_code;
 
-static void init_Linux_version(void) __attribute__((constructor));
+static void init_Linux_version(void) __attribute__((constructor(100)));
 static void init_Linux_version(void) {
     static struct utsname uts;
     int x = 0, y = 0, z = 0;	/* cleared in case sscanf() < 3 */
